uss-api/swagger.yaml

```diff
@@ -55,6 +55,9 @@ paths:
       tags:
```

Comment:
Github won't let me make a comment above this line... see lines 49-51.  Suggest removing the default security block since we are specifying per endpoint and that scope might be deprecated anyway.

Reply:
this is a bug.  Removed section.